frontend/src/service/notificationUserService.js

```diff
@@ -6,25 +6,25 @@ const API_getAllNotificationUser = () => {
   return axios
     .get('/api/user', createHeader(localStorage.getItem('JWT')))
     .then(response => response.data)
-    .catch(err => showError(err.message, 'Cant fetch Notification User Data.'))
+    .catch(err => showError(err.message, 'Can´t fetch Notification User Data.'))
```

Comment:
wouldn't it be more sensible to send the err.message you want to show from the backend directly instead of creating a new one here?